packages/calling-component-bindings/src/utils/videoGalleryUtils.ts

```diff
@@ -82,14 +93,24 @@ export const convertRemoteParticipantToVideoGalleryRemoteParticipant = (
     }
   }
 
+  /**
+   * Since VideoTile only cares about the following states,
+   * we don't observe the other states like 'EarlyMedia'.
+   */
+  let participantState: VideoGalleryRemoteParticipantState | undefined;
+  if (state === 'Connecting' || state === 'Ringing' || state === 'Hold') {
```

Comment:
Per comment on other PR, I think we should support 'Connected' as a valid state and pass that through

Reply:
Any particular reason for supporting the Connected state?
There is no visual feedback to the customers when the state is connected